app.js

```diff
@@ -1,8 +1,11 @@
 var path = require('path');
 var express = require('express');
+var bodyParser = require('body-parser');
 
 var app = express();
 
+var settings = require('./app/controller/settings');
```

Comment:
this should not bee here. this should be required in your routes file and the routes file should be required here